eng/Directory.Build.Data.props

```diff
@@ -52,6 +52,7 @@
     <UpdateSourceOnBuild Condition="'$(UpdateSourceOnBuild)' == ''">$(AZURE_DEV_UPDATESOURCESONBUILD)</UpdateSourceOnBuild>
     <PowerShellExe Condition="'$(PowerShellExe)' == ''">pwsh</PowerShellExe>
     <CoverletOutputFormat Condition="'$(CoverletOutputFormat)' == '' and '$(CollectCoverage)' == 'true'">cobertura</CoverletOutputFormat>
+    <InheritDocEnabled Condition="'$(EnableInheritDoc)' == '' or '$(EnableInheritDoc)' == 'true'">true</InheritDocEnabled>
```

Comment:
Maybe just `'$(EnableInheritDoc)' != 'false'`? Also, why not just use a single variable for whether it's enabled or not. Seems a lot less to manage overtime (and easier to diagnose why something isn't working as expected.

Reply:
Your suggestion removed the reason I did two variables. I didn't want to duplicate the `or` logic everywhere.